flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/CliUtils.java

```diff
@@ -109,7 +109,9 @@ public static boolean createFile(final Path filePath) {
             if (parent == null) {
                 return false;
             }
-            Files.createDirectories(parent);
+            if (Files.notExists(parent)) {
```

Comment:
I reverted this change and re-run the new tests, but the tests are still passed. So the tests can't reproduce the problems mentioned in the JIRA issue. Could you update tests to reproduce your problem?

Reply:
ok, I will update the test case.